proposals/001-kroxylicious-operator-api-v1alpha.md

```diff
@@ -289,23 +277,28 @@ spec:
     selectorConfig: {}
 ```
 
-## KafkaClusterRef
+## KafkaService
 
-KafkaClusterRef points to a Kafka cluster. It might be a Kafka cluster stood up on the same Kubernetes cluster, a cluster
-running on a remote Kafka cluster, it might be service running on bare metal, or a Kafka service of a Cloud Provider.
+KafkaService points to a Kafka cluster. It might be a Kafka cluster stood up on the same Kubernetes cluster, a Kafka cluster
+running on a remote Kubernetes cluster, it might be service running on bare metal, or a Kafka service of a Cloud Provider.
```

Comment:
```suggestion
KafkaService represents, and encapsulates connection information for, a Kafka cluster. It might be a Kafka cluster stood up on the same Kubernetes cluster, a Kafka cluster
running on a remote Kubernetes cluster, it might be service running on bare metal, or a Kafka service of a Cloud Provider.
```